AltStore/LaunchViewController.swift

```diff
@@ -77,17 +84,12 @@ class LaunchViewController: RSTLaunchViewController
     }
     
     func displayError(_ msg: String) {
-        let label = UILabel()
-        label.text = msg
-        label.textColor = .refreshRed
-        label.sizeToFit()
-        
-        self.view.addSubview(label)
-        
-        NSLayoutConstraint.activate([
-            label.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            label.centerYAnchor.constraint(equalTo: view.centerYAnchor)
-        ])
+        print(msg)
```

Comment:
What error messages does this affect? Does this replace the standard blue pop-ups on the bottom? Does it replace *every* error message?

Reply:
Joe put this in, but it was broken (it showed red text up in the top notch). I didn’t change the normal error messages.

Reply:
Ah. Would it be possible to hook into the normal error messages? Either make them look like this, or make this use them? Just for consistency

Reply:
It uses the same UI alert that shows when it can't save to the app group